packages/slo-stat-bot/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "slo-stat-label",
+  "version": "0.0.1",
+  "description": "Labels SLO status on Issues and PRs",
+  "private": true,
+  "author": "Google LLC.",
+  "license": "Apache-2.0",
+  "repository": "https://github.com/googleapis/repo-automation-bots.git",
+  "homepage": "https://github.com/googleapis/repo-automation-bots",
+  "bugs": "https://github.com/googleapis/repo-automation-bots/issues",
+  "main": "build/src/app.js",
+  "keywords": [
+    "slo",
+    "probot",
+    "github",
+    "probot-app"
+  ],
+  "files": [
+    "build/src",
+    "build/data"
+  ],
+  "scripts": {
+    "compile": "tsc -p .",
+    "start": "probot run ./build/src/slo-stat-label.js",
+    "start:local": "node ./build/src/local.js",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit build/test",
+    "test:snap": "SNAPSHOT_UPDATE=1 npm test",
+    "fix": "gts fix",
+    "lint": "gts check"
+  },
+  "dependencies": {
+    "ajv": "^6.12.2",
+    "gcf-utils": "1.6.2",
```

Comment:
Could we try using the latest version of `gcf-utils`, it's at 5.x.x.

If the update isn't trivial, please make a tracking issue, and we can upgrade later (the benefit of the latest version, is that it retries on failure).

Reply:
Yes for sure! I will upgrade the gcf-utils to the most recent version